packages/ckeditor5-utils/src/language.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module utils/language
+ */
+
+const RTL_LANGUAGE_CODES = [
```

Comment:
Are the 3-letter versions in the standard? In the UI we support the 2-letter ones only, AFAIK.

Reply:
3-letter version is supported by CKE4.